compiler/q-implant/README.md

```diff
@@ -2,13 +2,21 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
```

Comment:
(maybe in another PR)

from https://github.com/Samsung/ONE/pull/11579#discussion_r1338074701

how about using `implant` or `embed` instead of `write` ?

Reply:
I thought the expression "write" was good in terms of overwriting, 
but I think the expression "implant" you suggested is better. 
I'll fix it.
Thank you!